tests/auth_tests/test_auth_backends.py

```diff
@@ -667,3 +667,48 @@ def test_backend_path_login_with_explicit_backends(self):
         user = User.objects.create_user(self.username, 'email', self.password)
         self.client._login(user, self.other_backend)
         self.assertBackendInSession(self.other_backend)
+
+
+@override_settings(
+    AUTHENTICATION_BACKENDS=[
+        'django.contrib.auth.backends.AllowInactiveUsersModelBackend'
+    ],
+)
+class AllowInactiveUsersModelBackendTest(TestCase):
+    """
+    To verify that the login form rejects inactive users, use an
+    authentication backend that allows them.
+    """
+    @classmethod
+    def setUpTestData(cls):
+        cls.user = User.objects.create_user(
+            username='test',
+            email='test@example.com',
+            password='test'
+        )
+        cls.user.is_active = False
+        cls.user.save()
+
+    def test_authenticate(self):
+        self.assertEqual(
+            authenticate(
+                username='test',
+                email='test@example.com',
+                password='test'
+            ),
+            self.user
+        )
+
+    def test_get_user(self):
```

Comment:
`test_authenticate` can be part of #6090, we'll only merge #6124 after #6090 is merged so `test_get_user` could be part of that patch. Actually could we change `test_get_user` to use `self.client.force_login(self.user)` and include both tests in #6090?